### PR TITLE
updated version of AliCEPUtils::CaloClusterTrackdmax

### PR DESCRIPTION
**Pull request by pbuehler**
. checks if every EMC cluster has a matching track

### CODE REVIEW

File: PWG/EMCAL/README.txt [hunk 1]
```diff
@@ -11,7 +11,8 @@ to steer the different steps in the preparation chain, handle data, process
 trigger quantities, and combine common functionalities. Furthermore helper
 classes for various issues are provided.
 
-It takes care of the EMCAL cluster correction, of the EMCal triggers, and of the jet reconstruction.
+It takes care of the EMCAL cluster correction, of the EMCal triggers, and of the
+jet reconstruction.
```
Comment (raymondEhlers) [inline PWG/EMCAL/README.txt]: What's the purpose of this change?

**Comment by raymondEhlers** [comment]
+test

File: PWG/EMCAL/README.txt (hunk 1 above)
Comment (pbuehler) [reply PWG/EMCAL/README.txt]: this change in the README.txt files was not made on purpose. It probably happened when I scrolled through the text. By mistake a "newline" was introduced. I'm not sure how to resolve this. Can I cancel the merge request and then repeat the procedure without the changes in the README.txt file.

Comment (raymondEhlers) [reply PWG/EMCAL/README.txt]: Okay, no problem. To fix it, you should:

```
$ git reset --soft HEAD~1                                   # Undo your commit, but keep the files (--soft is important!)
$ git checkout -- PWG/EMCAL/README.txt    # Undo the changes to the README
$ git commit                                                          # Recreate your commit
$ git push --force  (name of your remote)        # Push the new commit to your fork's master, which is the branch you used for this pull request.
```

this will update the current pull request (PR) with your new commit without changes to the EMCal README

Comment (pbuehler) [reply PWG/EMCAL/README.txt]: Sorry Raymond that I bother you with this.
"git reset --soft HEAD~1" says "fatal: Cannot do a soft reset in the middle of a merge."
Any suggestion?
